tests/address_bar_and_search/test_search_term_persists.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from selenium.webdriver import Firefox
+
+from modules.browser_object import Navigation, TabBar
+from modules.page_object import AboutConfig
+
+
+# Set search region
+@pytest.fixture()
+def add_prefs():
+    return [
+        ("browser.search.region", "US"),
+        ("browser.urlbar.showSearchTerms.enabled", True),
+        ("browser.urlbar.showSearchTerms.featureGate", True),
+    ]
+
+
+# Set constants
+FIRST_SEARCH = "cheetah"
+FIRST_RESULT = "https://www.google.com/search?client=firefox-b-1-d&q=cheetah"
+SECOND_SEARCH = "lion"
+SECOND_RESULT = "https://www.google.com/search?client=firefox-b-1-d&q=lion"
+
+
+def test_search_term_persists(driver: Firefox):
+    """
+    C2153943 - Persist search term basic functionality
+    """
+
+    # Create objects
+    nav = Navigation(driver).open()
+    tab = TabBar(driver)
+
+    def toggle_old_search_bar():
+        tab.new_tab_by_button()
+        window_handles = driver.window_handles
+        driver.switch_to.window(window_handles[-1])
+        ac = AboutConfig(driver)
+        pref = "browser.search.widget.inNavBar"
+        ac.toggle_true_false_config(pref)
+        nav.set_chrome_context()
+        x_icon = tab.get_element("tab-x-icon", multiple=True)
+        x_icon[1].click()
```

Comment:
Can you add a comment on why its the 2nd x icon that you are looking for? It isn't too clear at first glance